src/modules/70-pipeline/pages/execution/ExecutionPolicyEvaluationsView/__tests__/ExecutionPolicyEvaluationsView.test.tsx

```diff
@@ -11,6 +11,15 @@ import { TestWrapper } from '@common/utils/testUtils'
 import mockImport from 'framework/utils/mockImport'
 import ExecutionPolicyEvaluationsView from '../ExecutionPolicyEvaluationsView'
 
+jest.mock('services/pm', () => ({
+  useGetEvaluationList: jest.fn(() => ({
+    data: null,
+    loading: false,
+    refetch: jest.fn().mockReturnValue({}),
```

Comment:
Does this get called at all? If not, you can probably simplify
```suggestion
    refetch: jest.fn(),
```